### PR TITLE
chore: validate images from registry via Pepr (ADR)

### PR DESCRIPTION
**Pull request by btlghrants**
## Description

Working through _how_ we might accomplish validating image signatures against pubkeys derived from the OCI registries that serve them.

Relates to https://github.com/defenseunicorns/pepr/issues/1240

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging
- [x] Unit, [Journey](https://github.com/defenseunicorns/pepr/tree/main/journey), [E2E Tests](https://github.com/defenseunicorns/pepr-excellent-examples), [docs](https://github.com/defenseunicorns/pepr/tree/main/docs), [adr](https://github.com/defenseunicorns/pepr/tree/main/adr) added or updated as needed
- [x] [Contributor Guide Steps](https://docs.pepr.dev/main/contribute/#submitting-a-pull-request) followed

### CODE REVIEW

File: adr/0014-admission-verify-img-sigs.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+The Go library could be shipped along with the Pepr controller & Pepr could access it through using a cross-language access technology (i.e. WebAssembly).
+
+##### Pros
+
+- Requires minimal new Pepr-custom code
+
+- Lib code from repo under the sigstore umbrella
+
+- WebAssembly is neat!
```
Comment (cmwylie19) [inline adr/0014-admission-verify-img-sigs.md:103]: haha!

File: adr/0014-admission-verify-img-sigs.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+##### Pros
+
+- Written in Pepr-native language (JavaScript) so fits into existing toolset
+
+- Lib code from repo under the sigstore umbrella
+
+##### Cons
+
+- n/a
```
Comment (cmwylie19) [inline adr/0014-admission-verify-img-sigs.md]: We are pretty sure but not 100% yet right?

File: adr/0014-admission-verify-img-sigs.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+### Chosen Implementation Option
+
+Install JS-native lib &mdash; [sigstore-js](https://github.com/sigstore/sigstore-js).
+
+Despite the other options having more appeal in terms of technical interest (WebAssembly is _neat_!) and/or offering to take Pepr a step twoard other "cloud native" tools (i.e. largely available as Go libs), this option is:
```
Comment (cmwylie19) [inline adr/0014-admission-verify-img-sigs.md]:
```suggestion
Despite the other options having more appeal in terms of technical interest (WebAssembly is _neat_!) and/or offering to take Pepr a step toward other "cloud native" tools (i.e. largely available as Go libs), this option is:
```

**Review by cmwylie19 — CHANGES_REQUESTED** [review]

File: adr/0014-admission-verify-img-sigs.md (hunk 2 above)
Comment (btlghrants) [reply adr/0014-admission-verify-img-sigs.md]: K, put some "hedge" words & a plan of action to validate in there.  Like that better?

**Review by cmwylie19 — APPROVED** [review]

**Pull request merged**